cpp/include/core/concepts/GameRules.hpp

```diff
@@ -13,12 +13,22 @@ template <typename GR, typename GameTypes, typename GameResultsTensor, typename
 concept GameRules = requires(const State& const_state, const StateHistory& const_history,
                              State& state, StateHistory& history, group::element_t sym,
                              core::seat_index_t last_player, core::action_t last_action,
-                             GameResultsTensor& results) {
+                             GameResultsTensor& results, action_mode_t action_mode) {
+  // Initialize the state.
   { GR::init_state(state) };
+  // Get the legal moves for the state history.
   { GR::get_legal_moves(const_history) } -> std::same_as<typename GameTypes::ActionMask>;
+  // Get the action mode for the state.
   { GR::get_action_mode(const_state) } -> std::same_as<core::action_mode_t>;
+  // Get the current player for the state.
```

Comment:
Remove this comment and add that `get_current_player()` assumes not chance mode.

Reply:
I realize that in `Node::stable_state_t` constructor, we call `get_current_player()` even when it's chance-mode. I hope that this value is never used for chance-nodes? Maybe add a comment to this effect in the `stable_state_t` constructor.